Bottom: 3e3724434caedf6f286d2f8dd54bdc4526664733
Top:    d86d0baadd9e76015234d9fe97477781a201c794
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-20 00:56:45 +0000

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index 0a7b68e..c6bc7de 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -23,6 +23,8 @@
 #ifndef __HIDGL_INCLUDED__
 #define __HIDGL_INCLUDED__
 
+#include "hidgl_shaders.h"
+
 #define TRIANGLE_ARRAY_SIZE 30000
 typedef struct {
   GLfloat *triangle_array;
@@ -39,6 +41,8 @@ typedef struct {
 extern triangle_buffer buffer;
 extern float global_depth;
 
+extern hidgl_shader *circular_program;
+
 void hidgl_in_context (bool is_in_context);
 void hidgl_init_triangle_array (triangle_buffer *buffer);
 void hidgl_finish_triangle_array (triangle_buffer *buffer);
